core-kotlin-modules/core-kotlin-advanced-4/pom.xml

```
@@ -13,6 +13,13 @@
         <version>1.0.0-SNAPSHOT</version>
     </parent>
 
+    <properties>
+        <kotlinx-html-jvm.version>0.7.5</kotlinx-html-jvm.version>
+        <kotlinx-coroutines.version>1.7.3</kotlinx-coroutines.version>
+        <kotlinx-coroutines.version>1.6.0</kotlinx-coroutines.version>
```

duplicate, only keeping the later one.

```suggestion
        <kotlinx-coroutines.version>1.7.3</kotlinx-coroutines.version>
```